src/dist/replication/lib/duplication/test/duplication_test_base.h

```diff
@@ -13,6 +13,9 @@
 namespace dsn {
 namespace replication {
 
+DEFINE_STORAGE_WRITE_RPC_CODE(RPC_DUPLICATION_IDEMPOTENT_WRITE, false, true)
```

Comment:
Suggest update definition like below which is clearer.
`DEFINE_STORAGE_WRITE_RPC_CODE(RPC_DUPLICATION_IDEMPOTENT_WRITE, NOT_ALLOW_BATCH, IS_IDEPOTENT)`

Reply:
Reasonable. We can do this refactoring for all our usages of `DEFINE_STORAGE_WRITE_RPC_CODE`, but later, not in this PR.